test/fixtures/dotnet-fs-simple-project/manifest.fsproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Controllers/ValuesController.fs" />

```

Comment:
hmm wonder how this Compile property plays a role in dependency resolution, do you know?

Reply:
I'm almost sure it is not related to dependencies.

Reply:
It looks more like paths to particular project files, maybe with some compiler directives...